website_team/README.rst

```diff
@@ -0,0 +1,43 @@
+=======
+ Website Team
+=======
+
```

Comment:
Use the next rule:
=============
 [Module name]
=============
correct length the =